services/core-api/app/api/projects/project_summary/models/project_summary.py

```diff
@@ -278,13 +279,25 @@ def create_or_update_party(cls, party_data, job_title_code, existing_party):
                 new_party.address.append(new_address)
             return new_party
 
-    def create_or_update_authorization(self, authorization, is_ams):
-            valid_authorization = ProjectSummaryAuthorization.validate_authorization(authorization, is_ams)
-            if valid_authorization != True:
-                current_app.logger.info(f'Validation failed for the following authorization with error {valid_authorization}')
-                current_app.logger.info(authorization)
-                raise BadRequest(valid_authorization)
-
+    @classmethod
+    def create_or_update_party_orgbook(cls, party_data, party_guid, existing_party):
```

Comment:
I think all of this logic was removed in a previous PR, so might want to revert this part.
PR: https://github.com/bcgov/mds/pull/3067/files

Reply:
Thanks, this has been removed now.